### PR TITLE
Update client docs with new endpoint source

### PR DESCRIPTION
**Pull request by abidwael**

### CODE REVIEW

File: docs/reference/python_client.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Example:
 ```python
 from lorax import Client
 
-endpoint_url = f"https://api.app.predibase.com/v1/llms/{llm_deployment_name}"
+endpoint_url = f"https://serving.app.predibase.com/{predibase_tenant_id}/llm/llm-deployment-{llm_deployment_name}/generate"
```
Comment (brightsparc) [inline docs/reference/python_client.md:70]: Can we remove the `llm-deployment-` when resolving the URL in the backend - we would need to update the VS for this too.

Comment (tgaddair) [reply docs/reference/python_client.md:70]: +1, less verbose the better.

Comment (tgaddair) [reply docs/reference/python_client.md:70]: Can you also explain how to obtain the tenant ID?

Comment (tgaddair) [reply docs/reference/python_client.md:70]: Also, we need to use `/` endpoint, not `/generate`.

Comment (abidwael) [reply docs/reference/python_client.md]: updated with information on the tenant short code. I'll need to update the VS to add `/` as a supported route

**Review by brightsparc — APPROVED** [review]

File: docs/reference/python_client.md [hunk 2]
```diff
@@ -67,7 +67,9 @@ Example:
 ```python
 from lorax import Client
 
-endpoint_url = f"https://api.app.predibase.com/v1/llms/{llm_deployment_name}"
+# You can get your Predibase API token by going to Settings > My Profile > Generate API Token
+# You can get your Predibase Tenant short code by going to Settings > My Profile > Overview > Tenant ID
+endpoint_url = f"https://serving.app.predibase.com/{predibase_tenant_short_code}/deployments/v2/llms/{llm_deployment_name}/generate"
```
Comment (tgaddair) [inline docs/reference/python_client.md:72]: This won't work with streaming endpoint, though. It really should be `/`, not `/generate`.

File: clients/python/lorax/client.py [hunk 3]
```diff
@@ -155,7 +155,9 @@ def generate(
         payload = resp.json()
         if resp.status_code != 200:
             raise parse_error(resp.status_code, payload)
-        return Response(**payload[0])
+        if isinstance(payload, list):
+            return Response(**payload[0])
```
Comment (tgaddair) [inline clients/python/lorax/client.py:159]: This smells like bad design IMO. The REST API should be consistent and predictable. We should not be writing client-side hacks to account for inconsistency in the REST API.

**Pull request merged**